test/unit/LivepeerETHTicketBroker.js

```diff
@@ -305,5 +306,29 @@ contract("LivepeerETHTicketBroker", accounts => {
                 assert.equal(events.length, 0)
             })
         })
+
+        describe("withdrawTransfer", () => {
```

Comment:
Should this describe block be nested within a parent describe block for `withdraw` similar to what is done for `redeemWinningTicket` and the nested describe blocks for `winningTicketTransfer` and `penaltyEscrowSlash`?

Reply:
👍